Polyhedron/demo/Polyhedron/MainWindow.cpp

```diff
@@ -1334,19 +1335,23 @@ void MainWindow::showSceneContextMenu(const QPoint& p) {
           CGAL::Three::Scene_item* item = scene->item(index);
           if(!item)
             continue;
-          QVector<QString> action_names;
-          Q_FOREACH(QAction* action, item->contextMenu()->actions())
-          {
-            action_names.push_back(action->text());
-          }
+          if(item->has_stats())
+            has_stats = true;
         }
         QMenu menu;
         Q_FOREACH(QString name, menu_actions.keys())
         {
           QAction* action = menu.addAction(name);
           connect(action, &QAction::triggered, this, &MainWindow::propagate_action);
         }
-
+        if(has_stats)
+        {
+          QAction* actionStatistics =
+              menu.addAction(tr("Statistics..."));
+          actionStatistics->setObjectName("actionStatisticsOnPolyhedron");
```

Comment:
The name of the action makes me think that it will only work with polyhedra, it is just the name of the action that is misleading?

Reply:
It is. Originally, only scene_polyhedron_items  had statistics. I didn't change the name of the action when I added stats for others.